src/django/api/models.py

```diff
@@ -1069,6 +1078,9 @@ def filter_by_query_params(self, params):
             FacilitiesQueryParams.BOUNDARY, None
         )
 
+        ppe = (True if params.get(FacilitiesQueryParams.PPE, '') == 'true'
```

Comment:
Confirming it's desired for `ppe` to only be `true` or `false` even though it is not required (in the API) and could be `None`. There may be no use case for `ppe == False` to return Facilities that don't have PPE and the value absent indicating that both PPE and non-PPE. I got a different understanding from the UI checkbox (binary) and the Swagger value selector (ternary), but that False and None give the same results.

Reply:
I did discuss this with the team and we came to the conclusion that a search that excludes PPE facilities was not likely to be helpful given that the PPE feature set is designed to help people find PPE producers.

I will attempt to address this ambiguity in states with an inline comment and some additional clarification in the swagger help text.

Reply:
Added in fixup 0bf9260